packages/vite/src/node/plugins/css.ts

```diff
@@ -898,9 +898,11 @@ async function doUrlReplace(
 }
 
 async function minifyCSS(css: string, config: ResolvedConfig) {
+  const target = config.build.target
   const { code, warnings } = await transform(css, {
     loader: 'css',
-    minify: true
+    minify: true,
+    target: target || undefined
```

Comment:
Why not directly `target: config.build.target`?

Reply:
Because it can be `false`, that is a Vite specific option

Reply:
Oh, but it could be directly `config.build.target || undefined` if that is what you are proposing 👍🏼

Reply:
Yes, it is better to use it directly, I have changed it, thanks.